test/end_to_end/appdefined_SUITE.erl

```diff
@@ -1,7 +1,6 @@
 -module(appdefined_SUITE).
--include_lib("common_test/include/ct.hrl").
```

Comment:
I am surprised that this include is not needed??

Reply:
So was I, but I don't really use any of the inbuilt ct functionality - there might be a long term task to use it more in the future.